contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -230,12 +270,20 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
     address from,
     uint256 id,
     bytes memory performData,
-    bool useTxGasPrice
+    bool isExecution
   ) internal view returns (PerformParams memory) {
     uint256 gasLimit = s_upkeep[id].executeGas;
     (uint256 gasWei, uint256 linkEth) = _getFeedData();
-    uint256 adjustedGasWei = _adjustGasPrice(gasWei, useTxGasPrice);
-    uint96 maxLinkPayment = _calculatePaymentAmount(gasLimit, adjustedGasWei, linkEth);
+    uint256 adjustedGasWei = _adjustGasPrice(gasWei, isExecution);
+    bytes memory data = new bytes(0);
+    if (PAYMENT_MODEL == PaymentModel.OPTIMISM) {
+      if (isExecution) {
+        data = bytes.concat(msg.data, L1_FEE_DATA_PADDING);
+      } else {
+        data = bytes.concat(performData, PERFORM_DATA_PADDING);
```

Comment:
an idea to simplify here - maybe we can use `MAX_MSG_DATA` directly instead of slightly more accurate performData + padding? 

I'm thinking it will simplify code by removing one variable, and even though it will be slightly less accurate estimate, it's fine since it's within checkUpkeep and get's multiplied by multiplier anyway.

wdyt?

Reply:
Good idea. Good to reduce hardcoded bytes vars